api/it/database/database.go

```diff
@@ -32,7 +32,7 @@ func create(conn *sql.DB, dbName string) (*sql.DB, error) {
 			log.Fatalf("Failed to cleanup integration test database: \n%s", err)
 		}
 		return nil, err
-	} else {
+	} else { //nolint:revive // https://github.com/mgechev/revive/issues/564 (false positive)
```

Comment:
This is needed because of a known issue of the linter to flag a false positive of an undeclared variable when it's a variable that's short-assigned (declared and assigned with `:=`) in an `if` block.

Reply:
will it be still error if we remove else just call return?

Reply:
Hmm yeah I've refactored both of those code blocks where I added that linter exception line, so there's no need for nested if-else blocks (and the linter error stops appearing also). Hopefully it's much easier to read also.